src/test/kotlin/com/workos/test/auditlogs/AuditLogsTest.kt

```diff
@@ -247,4 +248,139 @@ class AuditLogsTest : TestBase() {
       assert(exception.errors?.size == 1)
     }
   }
+
+  fun createExportShouldNotThrowException() {
+    val workos = createWorkOSClient()
+
+    stubResponse(
+      url = "/audit_logs/exports",
+      responseBody = """{
+        "object": "audit_log_export",
+        "id": "audit_log_export_123",
+        "state": "pending",
+        "created_at": "2022-08-17T19:58:50.686Z",
+        "update_at": "2022-08-17T19:58:50.686Z"
```

Comment:
```suggestion
        "updated_at": "2022-08-17T19:58:50.686Z"
```

Reply:
Weird -- does this test not fail?

Reply:
Hmm, yeah odd. I'll have to look into why the test passes.

Reply:
@dliu-workos the test function was missing the `@Test` decorator.